src/problem4.js

```diff
@@ -1,6 +1,32 @@
 function problem4(word) {
-  var answer;
-  return answer;
+  let string = '';
+
+  for (let i = 0; i < word.length; i += 1) {
+    let char = word.charAt(i);
+
+    const isAlphabet = checkAlphabet(char);
+    if (isAlphabet) {
```

Comment:
```suggestion
    if (checkAlphabet(char)) {
```
사소하지만, 함수의 리턴값이 확실하게 예상되므로 불필요한 선언을 줄일 수 있을 것 같아요. 함수명을 `isAlphabet`처럼 바꾸는 것도 고려할 만 해보이네요.

Reply:
너무 과도하게 설명적이었던 것 같기도 하네요.
개인적으로 `is` 형태의 네이밍은 boolean 형 변수에만 쓰고 있습니다.
boolean을 반환하는 함수는 `check`와 같은 네이밍을 사용하는데요.
그 이유는 변수에도 is를 쓰고 함수에도 is를 쓰면 코드를 읽을 때는 파라미터의 존재 유무로 변수인지 함수인지 확인이 쉬우나,
코드를 작성할때나 모듈을 불러올때는 이게 변수였나 함수였나 헷갈릴 때가 있더라구요.